cmds/exp/netbootxyz/netbootxyz.go

```diff
@@ -174,12 +173,15 @@ func (o OSEndpoint) Load() error {
 			fmt.Printf("With Kernel at %s\n", tmpPath+"vmlinuz")
 			fmt.Printf("With Initrd at %s\n", tmpPath+"initrd")
 			fmt.Printf("Commandline: %s\n", o.Commandline)
+
 			// Load Kernel and initrd
 			err = kexec.FileLoad(vmlinuz, initrd, o.Commandline)
```

Comment:
one last thing, it's preferred if you can do the 
if err := kexec.FileLoad(...); err != nil

thanks